content/page/getting-involved/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Getting involved
+menu:
+    main: 
+        weight: 5
+        params:
+            icon: code-icon
+
+comments: false
+---
+
+The easiest way to get in touch is probably through [Discord](https://discord.gg/C9bzWgNmqk), where we discuss research topics, job opportunities, open hardware, spiking neural network training and much more. We'd be delighted to have you join! 
+If you feel like contributing to ONM but you're not exactly sure how, here are some ideas to get you started:
+
+* Link an interesting open source repository to our [collection](https://github.com/open-neuromorphic/open-neuromorphic) so that others can find it too! This can be a framework / package that deals with neuromorphic things or a neat implementation for example! ONM is meant as a platform to showcase your code and others'!
+* Write a blog post together with the community. If you think you learned something useful that you'd like to share with the community, why not publish it on our website? You can have a look at the structure of [existing posts](https://github.com/open-neuromorphic/open-neuromorphic.github.io/tree/main/content/post) and open a PR for a new one. The community will then give you feedback to improve it if you wish to do so. Think about it as a mini paper!
```

Comment:
I would also add that an article reviewed by ONM community could be a nice addition also for their personal blog, so that they can "take advantage of us" as peer reviewers! This should be nice for them

Reply:
By that you mean that they post a draft of their article on Discord and then publish it wherever, is that correct?

Reply:
Uhm no, that's not what I had in mind. I was meaning cross-posting the article , i.e. on both their blog and ONM one, like I am doing